src/renderer/Renderer.ts

```diff
@@ -226,10 +226,17 @@ abstract class Renderer {
     const flicking = getFlickingAttached(this._flicking, "Renderer");
 
     const cameraElement = flicking.camera.element;
-    const cameraChilds = toArray(cameraElement.children);
+    const cameraChilds = toArray(cameraElement.childNodes);
```

Comment:
Did you change it because of the range of browser support?

Reply:
Well looks like you've selected only one commit in this PR.
That code is outdated and we're using `children` in the final commit.